pkg/mounter/nfs_common.go

```diff
@@ -0,0 +1,91 @@
+package mounter
+
+import (
+	"fmt"
+)
+
+const pwxtag = "Portworx Network Drive"
+const workDir = "C:\\var\\pxvol"
```

Comment:
This should ideally be the working dir passed at install time as a argument to driver.